src/urllib3/response.py

```diff
@@ -298,9 +459,6 @@ def data(self):
     def connection(self):
         return self._connection
 
-    def isclosed(self):
-        return is_fp_closed(self._fp)
```

Comment:
Did you intend to stop using `is_fp_closed` here? `close()` does not even set `self.closed` to True

Reply:
I meant to revert this, thanks for catching!